crates/crypto/src/lib.rs

```diff
@@ -45,7 +45,7 @@ mod tests {
     #[test]
     fn test_schnorr_signature_pass() {
         let secp = Secp256k1::new();
-        let mut rng = rand::thread_rng();
+        let mut rng = rand::rngs::OsRng;
```

Comment:
```suggestion
        let mut rng = OsRng;
```
same as above, do we need the `let mut rng`?